### ISSUE TITLE
Crossrunner crashing and Npcap installation

### ISSUE THREAD

**Issue by GerrySchultz** [ibrumby/Crossrunner-Issues#12]
Finally got the TCP/IP NDA installed, working and configured using the Uthernet II driver.  I set the DNS and local IP addresses in the Setup Connection (Marinetti 3.0b8) as well as set the LAN slot to 2.  In the edit configuration of slot 2 is set to Uthernet II but in the "Edit Uthernet II Configuration" Adapter dialog box, there is nothing there and nothing to select from.  The Apple IIgs image starts up and runs fine until I push the "Connect to network" button in the TCP/IP NDA and then Crossrunner crashes and shuts down.  BTW, I installed Npcap 1.78 and selected to install "Support raw 802.11 traffic" as shown in my attachment.  So it appears to be a bug but if I need more information I'd appreciate it. 

Thank you for the help,
Gerry
![Npcap](https://github.com/ibrumby/Crossrunner-Issues/assets/152728620/5e07fb66-11c9-4386-88ba-e2b5666949ef)

**Comment by GerrySchultz** [comment]
Reinstalled Npcap 1.78 and added "Restrict Npcap driver's access to Admins only" but no change in symptoms.  Also contacted  Ewen Wannop in Great Britain.  He wrote the Sweet16 Starter Kit as well as the Sweet16 link layer.  Here's his response:

Gerry,

Hi Ewen, it’s been a long time since we’ve spoken and thank you for writing Webber.  The reason I’m writing you is, in case you’re not aware, there is a new Apple IIgs emulator that has been just released.  It’s called Crossrunner IIgs and it’s designed to run on the latest Windows PCs.  I got it up and running by talking to the people at Slack.  Ken Gagne sent me a link https://www.kansasfest.org/slack/

Anyway I was writing you as you made the Sweet16 Starter Kit as well as support Marinetti that a starter kit for Crossrunner would be very helpful.  I’m sure you are busy so this is just a suggestion.

----------

I have never owned a Windows computer, and do not have access to one, so have no idea what considerations would be needed to build a suitable starter kit for it. I would suspect though that if Crossrunner is a true IIgs emulator, then most applications should work just fine. The exceptions would be software that either uses the serial port, or needs a custom Link Layer for Marinetti.

The serial port generally is not available in most emulators, though Crossrunner may make an emulated port available. In which case software should not need to be changed. As to a Link Layer for Marinetti to work in emulators, I have only worked on the Sweet16 Link Layer, and that uses the COP vector in the emulated 65C816 chip. Sweet16 receives calls to that vector and internally processes the Ethernet packets that Marinetti works with, and feeds them back and forth into the Mac’s TCP/IP stream…

To sum up, I don’t think I can really help you, as you would need someone with a working Crossrunner emulation to build and check not only a suitable starter disk, but then build a custom Link Layer for Marinetti.

I have the source code for the generic Link Layer that was used to build the custom Sweet16 Link Layer. I can always provide a copy of that to a developer with the right equipment and knowledge to build the required Link Layer for Crossrunner.

Regards - Ewen
